tests/df/test_df_create.py

```diff
@@ -0,0 +1,76 @@
+import io
+import pathlib
+import tempfile
+import contextlib
+
+from dffml.cli import Dataflow
+from dffml.util.asynctestcase import AsyncTestCase
+from dffml import (
+    DataFlow,
+    Definition,
+    Input,
+    op,
+    ConfigLoaders,
+    MemoryOrchestrator,
+    GetSingle,
+)
+
+
+@op(
+    inputs={"input_string": Definition(name="InputString", primitive="str")},
+    outputs={
+        "output_string": Definition(name="OutputString", primitive="str")
+    },
+)
+def echo_string(input_string):
+    return {"output_string": input_string}
```

Comment:
Let's make it so that this testcase makes a temporary directory, writes this operation into it as the file `ops.py` or something, and then runs create. This way we verify that `relative=True` was added everywhere it was needed